src/Metadata/MetadataAwareMetadataBase.php

```diff
@@ -3,12 +3,37 @@
 namespace Tuf\Metadata;
 
 use Tuf\Exception\MetadataException;
+use Tuf\Exception\PotentialAttackException\RollbackAttackException;
 
 /**
- * Common methods for metadata that contain information about other metadata objects.
+ * Base class for metadata that contains information about other metadata.
  */
-trait MetaFileInfoTrait
+abstract class MetadataAwareMetadataBase extends MetadataBase
```

Comment:
Wonder if we should nave this `FileMetaAwareMetadataBase` because the actual item in the JSON is `meta` and all of the formats are "metadata aware"

Reply:
Yeah, I wasn't a huge fan of the name either. I don't know how I feel about `FileMeta` though; that's such a strange turn of phrase. I was thinking `MetadataInspectorBase`, but I will bring this up in scrum to see if Travis or Alex have some brilliant idea.